### PR TITLE
dev/reference/configuration: add a warning sentence

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 
Add a warning sentence

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/2075
### Which version does your change affect? 

dev

### CODE REVIEW

**Comment by CaitinChen** [comment]
@ichn-hu @TomShawn PTAL

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 1]
```diff
@@ -84,7 +84,9 @@ Configuration items related to log.
 
 > **Note:**
 >
-> Backward compatibility is supported and the initial `disable-timestamp` configuration item remains valid. But if the value of `disable-timestamp` semantically conflicts with the value of `enable-timestamp` (for example, if both `enable-timestamp` and `disable-timestamp` are set to `true`), TiDB reports an error and refuses to start . Discard `disable-timestamp` and use `enable-timestamp` which is semantically easier to understand.
+> Backward compatibility is supported and the initial `disable-timestamp` configuration item remains valid. But if the value of `disable-timestamp` semantically conflicts with the value of `enable-timestamp` (for example, if both `enable-timestamp` and `disable-timestamp` are set to `true`), TiDB ignores the value for `disable-timestamp`. In later versions, the `disable-timestamp` configuration will be removed.
```
Comment (ichn-hu) [inline dev/reference/configuration/tidb-server/configuration-file.md]: I suggest "In order to be backward compatible the initial ...."

**Comment by CaitinChen** [comment]
@ichn-hu Updated. Thanks. PTAL

**Comment by ichn-hu** [comment]
LGTM

**Pull request merged**